pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: "Integrating MailChimp with your Node.js App"
+date: "2021-06-01"
+author: "Yashovardhan Agrawal"
+pullQuote: "A step by step guide of integrating your MailChimp Audiences with your Node.js app"
+image: "integrating-mailchimp-with-your-nodejs-app/mc4.png"
+tags: [connections, engineering]
+---
+
+# Integrating MailChimp with your Node.js App
+
+If you're reading this blog right now, the chances are that you're finding your way to add a MailChimp integration into your Node.js application. Don't worry; we have you covered! This is a step by step guide that will help you build a sample Node.js backend for your webpage and help you integrate it with the MailChimp API.
+This tutorial will discuss all the different things you need to look for within MailChimp with references to their excellent API documentation. We'll also talk about an alternative method using Grouparoo, which will do all the heavy lifting for you!
+
+## Getting started
+
+To get started, you'll be needing the following things prepared:
+
+- A MailChimp Account
+- Node.js & npm installed on your machine
+- A basic frontend application to send requests to your Node.js application.
```

Comment:
Links here would be helpful. Mailchimp is self-explanatory, but I usually like to have a reference to what I'm asking readers to accomplish. In this case, links to docs on how to install Node and NPM would be helpful.

For the front end app, that could be intimidating to some folks. Maybe they already have a front end app and that's okay. If not, it might be nice to point to something you'd recommend starting with, like Next.js.

Reply:
I'm including a very simple frontend example from w3schools.

Reply:
added links for everything.